packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -170,6 +177,86 @@ export function nft_transfer(token_id: string, bidder: string): void {
     account_media.set(design.prev_owner, accountMedia)
 }
 
+/* Approval system */
+
+export function nft_approve(token_id: string, account_id: string, msg: string|null = null): void {
+    assert_at_least_one_yocto();
```

Comment:
Why is `assert_one_yocto` not enough here?

Reply:
I'm not completely sure but I used the Rust example as a reference

Reply:
Got it!